aws/PAS-AIO-DR-template.yaml

```diff
@@ -0,0 +1,1227 @@
+                  print ''file copied successfully''
+                  return cfnresponse.send(event, context, cfnresponse.SUCCESS, {}, physicalResourceId)
+          
+              except Exception as E:
+                  print E
+                  return cfnresponse.send(event, context, cfnresponse.FAILED, {}, physicalResourceId)'
+      Runtime: python2.7
+      Handler: index.lambda_handler
+      Role: !GetAtt 'LambdaDeployRole.Arn'
+  StoreMasterPassword:
+    Type: AWS::CloudFormation::CustomResource
+    Version: '1.0'
+    Properties:
+      ServiceToken: !GetAtt 'StorePasswordLambda.Arn'
+      Password: !Ref 'VaultMasterPassword'
+    Condition: EULACondition
+    DependsOn:
+      - LambdaDeployRole
+  StoreAdminPassword:
+    Type: AWS::CloudFormation::CustomResource
+    Properties:
+      ServiceToken: !GetAtt 'StorePasswordLambda.Arn'
+      Password: !Ref 'VaultAdminPassword'
+    Condition: EULACondition
+    DependsOn:
+      - LambdaDeployRole
+  StoreDRPassword:
+    Type: 'AWS::CloudFormation::CustomResource'
+    Properties:
+      ServiceToken: !GetAtt StorePasswordLambda.Arn
+      Password: !Ref VaultDRPassword
+    DependsOn:
+      - LambdaDeployRole
+  CleanMasterPassword:
+    Type: AWS::CloudFormation::CustomResource
+    Version: '1.0'
+    Properties:
+      ServiceToken: !GetAtt 'DeletePasswordLambda.Arn'
+      key: !GetAtt 'StoreMasterPassword.SsmId'
+    Condition: EULACondition
+    DependsOn:
+      - LambdaDeployRole
+      - StorePasswordLambda
+      - VaultDRMachine
+  CleanAdminPassword:
+    Type: AWS::CloudFormation::CustomResource
+    Version: '1.0'
+    Properties:
+      ServiceToken: !GetAtt 'DeletePasswordLambda.Arn'
+      key: !GetAtt 'StoreAdminPassword.SsmId'
+    Condition: EULACondition
+    DependsOn:
+      - LambdaDeployRole
+      - StorePasswordLambda
+      - ComponentsMachine
+  CleanDRPassword:
+    Type: 'AWS::CloudFormation::CustomResource'
+    Version: '1.0'
+    Properties:
+      ServiceToken: !GetAtt DeletePasswordLambda.Arn
+      key: !GetAtt StoreDRPassword.SsmId
+    DependsOn:
+      - LambdaDeployRole
+      - StorePasswordLambda
+      - VaultDRMachine
+  VaultMachine:
+    Type: AWS::EC2::Instance
+    Properties:
+      Tags:
+        - Key: Name
+          Value: !Ref 'VaultInstanceName'
+      SecurityGroupIds: !Ref 'VaultInstanceSecurityGroups'
+      SubnetId: !Ref 'VaultInstanceSubnetId'
+      ImageId: !FindInMap
+        - RegionMap
+        - !Ref 'AWS::Region'
+        - Vault
+      InstanceType: !Ref 'VaultInstanceType'
+      UserData: !Base64
+        Fn::Sub: |-
+          <script>
+          cfn-init.exe -v -s ${AWS::StackId} -r VaultMachine --configsets ascending --region ${AWS::Region}
+          </script>
+      KeyName: !Ref 'KeyName'
+      IamInstanceProfile: !Ref 'VaultInstancesProfile'
+    Metadata:
+      AWS::CloudFormation::Init:
+        configSets:
+          ascending:
+            - configSSMAndHostname
```

Comment:
please change the name of configset according to AIO deployment
we don't change the hostname so we can remove HostName from the name

Reply:
changed